dev/releases/release_notes.py

```diff
@@ -99,24 +96,25 @@ def warning(s):
     ["topic: polyhedral geometry",  "Polyhedral Geometry"],
     ["topic: toric geometry",       "Toric Geometry"],
     ["topic: tropical geometry",    "Tropical Geometry"],
-
-    ["serialization",               "Changes related to serializing data in the MRDI file format"],
-
-    ["enhancement",                 "New features or extended functionality"],
-    ["experimental",                "Only changes experimental parts of OSCAR"],
-    ["optimization",                "Performance improvements or improved testing"],
-    ["bug: crash",                  "Fixed bugs that could lead to crashes"],
-    ["bug",                         "Other fixed bugs"],
-    ["documentation",               "Improvements or additions to documentation"],
-
     ["package: AbstractAlgebra",    "Changes related to the package AbstractAlgebra"],
     ["package: AlgebraicSolving",   "Changes related to the package AlgebraicSolving"],
     ["package: GAP",                "Changes related to the package GAP"],
     ["package: Hecke",              "Changes related to the package Hecke"],
     ["package: Nemo",               "Changes related to the package Nemo"],
     ["package: Polymake",           "Changes related to the package Polymake"],
     ["package: Singular",           "Changes related to the package Singular"],
-
+]
+prtypes = [
+    ["renaming",                    "Renamings"],
+    ["serialization",               "Changes related to serializing data in the MRDI file format"],
+    ["enhancement",                 "New features or extended functionality"],
+    ["experimental",                "Only changes experimental parts of OSCAR"],
+    ["optimization",                "Performance improvements or improved testing"],
+    ["bug: crash",                  "Fixed bugs that could lead to crashes"],
+    ["bug",                         "Other fixed bugs"],
+    ["bug: wrong result",           "Fixed bugs that returned incorrect results"],
+    ["bug: unexpected error",       "Fixed bugs that resulted in unexpected errors"],
```

Comment:
I think "other fixed bugs" should be last; and "wrong results" first (they are the worst kind of bugs; crashes are 2nd worst so should be next)
```suggestion
    ["bug: wrong result",           "Fixed bugs that returned incorrect results"],
    ["bug: crash",                  "Fixed bugs that could lead to crashes"],
    ["bug: unexpected error",       "Fixed bugs that resulted in unexpected errors"],
    ["bug",                         "Other fixed bugs"],
```